Translate 2 docs into Japanese

clariroid opened the pull request:
Translate these docs into Japanese:  
  * react/docs/docs/getting-started.md => get-started.jp.md
  * react/docs/docs/tutorial.md => tutorial.jp.md

I have already submitted a CLA.

zpao commented on docs/docs/getting-started.jp.md:
```diff
@@ -0,0 +1,116 @@
+---
+id: getting-started-ja
```
Overall this looks great!

The only thing I see being an issue is the locale id. Correct me if I'm wrong, but you probably want `ja-JP`. We already have `zh-CN` and #3240 adds `ko-KR`. Let's update and make sure the filenames, ids, and links are correct.

I'm far from an expert here, I'd just like to make sure we're consistent.

clariroid commented:
Thank you for the review!  
I have renamed those files and fixed the locale id.  

  * getting-started.jp.md  ->  getting-started.ja-JP.md
  * tutorial.jp.md  ->  turorial.ja-JP.md

If there is another issue, please tell me!

zpao commented:
lgtm!
